src/resolversFactory.js

```diff
@@ -0,0 +1,36 @@
+var util = require('util');
+var http = require('http');
+
+function createWofPipResolver(url) {
```

Comment:
Renaming this parameter to baseUrl and removing line 5 avoid having two variables called url :)